packages/customer/src/models/customer-group-customer.ts

```diff
@@ -21,19 +21,27 @@ export default class CustomerGroupCustomer {
   @PrimaryKey({ columnType: "text" })
   id!: string
 
+  @Property({ columnType: "text" })
+  customer_id: string
+
+  @Property({ columnType: "text" })
+  customer_group_id: string
+
   @ManyToOne({
     entity: () => Customer,
-    fieldName: "customer__id",
+    fieldName: "customer_id",
     index: "IDX_customer_group_customer_customer_id",
+    nullable: true,
   })
-  customer: Customer
+  customer?: Customer
```

Comment:
todo: apply new conventions everywhere. here it should be typed as `Customer | null` since it is nullable